@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -715,7 +722,10 @@ export function compileTechniques(techniques: Technique[]) {
             if (isJsonExpr(value) && propertyName !== "kind") {
                 // "kind" is reserved.
                 try {
-                    technique[propertyName] = Expr.fromJSON(value);
+                    const expr = Expr.fromJSON(value).intern(mapView.exprPool);
```

Comment:
we need APIs to shirk the expr pool.

Reply:
`MapView.set theme` is good enough IMO.
`Expr` instances are directly derived from `Theme`